bee-node/src/entrynode/builder.rs

```diff
@@ -223,7 +223,10 @@ async fn initialize_autopeering(
     let mut peerstore_options = RocksDbPeerStoreConfigOptions::default();
     peerstore_options.create_if_missing(true);
     peerstore_options.create_missing_column_families(true);
-    let peerstore_cfg = RocksDbPeerStoreConfig::new(PEERSTORE_PATH, peerstore_options);
+    let peerstore_cfg = RocksDbPeerStoreConfig::new(
+        builder.config().autopeering_config.peer_storage_path(),
+        peerstore_options,
```

Comment:
Should this be renamed as well now?

Reply:
there we go :wink:

Reply:
I will keep it for now if you allow me, I have some node refactoring/cleaning coming up soon

Reply:
But yes, it should

Reply:
I'ld be happy I don't need to rename everything to `peerstorage` now.